modules/player/src/main/java/unnamed/mmo/modifiers/ModifierType.java

```diff
@@ -0,0 +1,33 @@
+package unnamed.mmo.modifiers;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+import unnamed.mmo.registry.Registry;
+import unnamed.mmo.registry.Resource;
+
+public record ModifierType(String id, double defaultValue) implements Resource {
+
+   public static final Codec<ModifierType> CODEC = RecordCodecBuilder.create(i -> i.group(
+            Codec.STRING.fieldOf("modifier").forGetter(ModifierType::id),
+            Codec.DOUBLE.fieldOf("defaultValue").forGetter(ModifierType::defaultValue)
+    ).apply(i, ModifierType::new));
+
+    public static final Registry<ModifierType> REGISTRY = Registry.codec("modifiers", CODEC);
+
+
+    public static boolean doesModifierExist(@NotNull String modifierId) {
+        return REGISTRY.get("starlight:" + modifierId) != null;
+    }
+
+    public static double getBaseValue(@NotNull String modifierId) {
+        return REGISTRY.required("starlight:" + modifierId).defaultValue;
+    }
+
+
+    @Override
+    public @NotNull NamespaceID namespace() {
+        return NamespaceID.from("starlight", id);
```

Comment:
Since you are specifying the namespace in the modifiers json file, you should not do it here. Also using namespaceid here generally would be more consistent with other modules

Reply:
If I don't specify namespace, I get a load error. In this case, how would I change it so it uses the namespace in the file?